src/en/docs/usage-basic-overview.md

```diff
@@ -5,7 +5,24 @@ priority: 2
+There is an "X" in the top left of the photo. Clicking this takes you back to the timeline view.
+- **Click on the heart.** Placing the mouse cursor over the photo, you will see two symbols: In the bottom-left of the photo is a outline of a ♥ and in the upper-right there is an &#9432;. Clicking the heart-outline changes it to a solid heart to indicate that the photo is one of your favourites. 
+On the top of the screen there is a `checkbox` with the text "Show only favourites" next to it and checking it will only show the photos that you have marked with the heart.
+- **Click on the &#9432;** to open up a panel on the right side of the screen where you can see some metadata for that photo, such as when the photo was taken, with which camera, its exposure settings etc. It also shows the _path to the album/folder_ where the photo is stored. Clicking on the album path allows you to see all the photos from that album. Furthermore you can also see which faces that have been identified and the location where it was taken (if known). At the bottom of the panel, you have the option to download this photo to your computer or create a link that can be shared with others. More on that later.
+
+## Left-hand navigation panel ##
+On the left-hand side of the screen you have more options of how to view your photos. These are:
+- **Timeline** - covered above.
+- **Albums** - allows you to view all photos that resides in an album on your server. In Photoview, an album is the same as a folder on your server. This means that you can have as many levels of albums as you wish and they get their names from the names of the folders in your filesystem. In album view, when you click the "i" on the top right of a photo, you will get a chance to set this photo as the _album cover photo_ of that album.
+In album view, you will also see a cog-symbol next right to the name of the album. Clicking this allows you to create a _share_, esentially a _link_ to that album that you can share with your friends. By copying the link and including it in an e-mail or some other messenging service you can invite your friends to have a look at those photos in that album. You can also choose to download the album to the computer you are using to view the photos. You have the option to download the photos in full resolution or different scaled-down versions.
+- **Places** - this presents you with a map where your photos were taken (provided that they have geocoding information, which they normally have if they are taken with a mobile phone). You can zoom in and out of the map by using the scroll-wheel of your mouse. As you scroll in, you can see that the level of information about the location is becoming more and more precise. Clicking one of the pictures allows you to see all the photos from that location.
```

Comment:
> **Places** - this presents you

insert here `(if the feature is enabled)`

> from that location.

... by using left and right arrows to switch between them